src/index.html

```diff
@@ -9,6 +9,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__brand">
+        <img
+          src="./images/logo.png"
+          alt="Logo MOYO"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
```

Comment:
let's make links more informative
```suggestion
            <a href="#apple" class="nav__link is-active">apple</a>
```